tick/survival/tests/model_coxreg_partial_lik_test.py

```diff
@@ -24,6 +24,22 @@ def test_ModelCoxRegPartialLik(self):
         model_spars.fit(csr_matrix(features), times, censoring)
         self.run_test_for_glm(model, model_spars)
 
+    def test_sccs_serialize_and_compare(self):
```

Comment:
you meant `test_coxreg_serialize_and_compare`?

Reply:
copy/pasta